### DISCUSSION TITLE
Switch device to send message

### DISCUSSION THREAD

**Discussion by ibroid** [dimaskiddo/go-whatsapp-multidevice-rest#50] [Q&A] [answered]
i try install the pre build version. but i confused when i try to login multi device and send a message. how i do switch a device so i can decide wich device is want to send a message.

**Comment by ibnux** [comment, marked as answer]
You need to save generated JWT and pair it with phone number

Use that JWT to send message

JWT A for phone A.   
JWT B for phone B

When you want to send message using phone B then use JWT B
